app/client/cluster_api_client.py

```diff
@@ -238,6 +240,11 @@ def _graceful_shutdown_slaves(self, body):
         )
         return response
 
+    def get_console_output(self, build_id: int, subjob_id: int, atom_id: int, max_lines: int=50, offset: int=0):
```

Comment:
Can we add docstrings explaining what the return type is and what the content is?